js/components/portal-dashboard/question-navigator.tsx

```diff
@@ -68,4 +86,18 @@ export class QuestionNavigator extends React.PureComponent<IProps> {
     }
     return false;
   }
+
+  private renderChevron = (cssClass: string) => {
+    return (
+      <div onClick={this.handleChevronClick} data-cy="show-hide-question-button">
+        {<ArrowIcon className={cssClass} />}
+      </div>
+    );
+  }
+
+  private handleChevronClick() {
```

Comment:
Write as arrow function to remove bind above:
`private handleChevronClick = () => {`